sysconfig/sysconfig.go

```diff
@@ -19,10 +19,17 @@
 
 package sysconfig
 
+// Options is the set of options used to configure the run system
 type Options struct {
-	// TODO: do we really want this kind of specific dir pointers
-	// or more general ones?
+	// CloudInitSrcDir is where to find the cloud-init data when installing it,
+	// i.e. in early boot install mode it could be something like
+	// filepath.Join(boot.InitramfsUbuntuSeedDir,"data")
 	CloudInitSrcDir string
+
+	// TargetDir is the root directory where to install configure
+	// data, i.e. for cloud-init during the initramfs it will be something like
+	// boot.InitramfsWritableDir
+	TargetDir string
```

Comment:
I would still call this TargetRootDir and rename other related variable accordingly

Reply:
do you want this in this PR, or can it be a followup?

Reply:
well actually considering that travis is not triggering anything I will just do this and the other thing now :-/